src/server/game/Entities/Player/Player.cpp

```diff
@@ -1081,6 +1081,11 @@ void Player::Update(uint32 p_time)
 
     UpdateAfkReport(now);
 
+    if (IsInCombat())
```

Comment:
I tried something with ```m_ThreatManager.getThreatList();``` in the meantime but it returned an empty list while in combat D: